commons/feature_toggles/feature_toggle_manager.py

```diff
@@ -0,0 +1,69 @@
+import os
+import json
+from time_tracker_api.security import current_user_email
+from azure.appconfiguration import AzureAppConfigurationClient
+
+
+class MSConfig:
```

Comment:
`MSConfig` It was because this class was used with MS variables. In this case is different, we could use something like `FeatureToggleConfig` or `FTConfig` for example.

Reply:
We have changed to `FeatureToggleConfig` because is more explicit